pynest/nest/tests/test_topology_gamma_kernel.py

```diff
@@ -0,0 +1,124 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test of events
+"""
+
+import numpy as np
+import scipy.stats as sts
+import unittest
+import nest
+import nest.topology as tp
+from collections import Counter
+
+
+@nest.check_stack
+class TopologyGammaKernelTestCase(unittest.TestCase):
+    """Tests of gamma kernel in topology"""
+
+    def test_targets(self):
+        """Object targets"""
+
+        nest.ResetKernel()
+
+        extent = [100., 100.]
+        npop = 5000
+        positions = np.random.uniform(-extent[0] / 2, extent[1] / 2, [npop, 2])
+        ncon = 10000
+        kappa = 4.
+        theta = 4.
+        radius = extent[0] / 2 * np.sqrt(2)
+        dx = .5
+        x = np.arange(0, radius + dx, dx)
+
+        ly1 = tp.CreateLayer({
+            'positions': [[0., 0.]],
+            'elements': 'iaf_psc_alpha',
+            'extent': extent,
+            'edge_wrap': True
+        })
+
+        ly2 = tp.CreateLayer({
+            'positions': positions.tolist(),
+            'elements': 'iaf_psc_alpha',
+            'extent': extent,
+            'edge_wrap': True
+        })
+
+        tp.ConnectLayers(ly1, ly2, projections={
+            'connection_type': 'divergent',
+            'allow_oversized_mask': True,
+            'mask': {
+                'circular': {
+                    'radius': radius
+                }
+            },
+            'kernel': {
+                'gamma': {
+                    'kappa': kappa,
+                    'theta': theta
+                }
+            },
+            'number_of_connections': ncon,
+            'allow_multapses': True,
+            'allow_autapses': True,
+        })
+
+        gamma1 = sts.gamma.pdf(x, a=kappa, scale=theta)
+        gamma_normed1 = gamma1 / float(np.sum(gamma1)) / dx
+
+        gamma2 = sts.gamma.pdf(x, a=kappa + 1, scale=theta)
+        gamma_normed2 = gamma2 / float(np.sum(gamma2)) / dx
+
+        center = tp.FindCenterElement(ly1)
+        tgts = tp.GetTargetNodes(center, ly2)[0]
+        d = tp.Distance(center, tgts)
+
+        h = np.histogram(d, bins=x)[0]
+        h_normed = h / float(np.sum(h)) / dx
+        difference = np.diff([gamma_normed2[:-1], h_normed], axis=0)
+        error_score = np.abs(np.median(difference))
+        # print('Error: %s' % error_score)
+        assert(error_score < 10e-5)
+
+        if False:
+            import pylab as pl
+            fig, ax = pl.subplots(1)
+            ax.step(x[:-1], h_normed, label='Constructed targets in NEST')
+            ax.step(x, gamma_normed1, label='Gamma PDF: $a = \kappa$')
+            ax.step(x, gamma_normed2, label='Gamma PDF: $a = \kappa + 1$')
+            ax.legend()
+            pl.show()
```

Comment:
As this block is only used for debugging, I suggest removing `if False` and either commenting the block out, or removing it completely.